release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -244,6 +244,17 @@ With this update, you can assign IP addresses from a MetalLB `IPAddressPool` res
 
 For more information about assigning IP addresses from an IP address pool to services and namespaces, see xref:../networking/metallb/metallb-configure-address-pools.adoc#nw-metallb-configure-address-pool_configure-metallb-address-pools[Configuring MetalLB address pools].
 
+[id="ocp-4-13-supporting-ocp-nodes-with-dual-port-nics"]
+==== Supporting {product-title} installation on nodes with dual-port NICs (Technology Preview)
+
+With this update, {product-title} cluster can be deployed on a bond interface with 2 virtual function (VFs) on 2 physical functions (PFs) using the following methods:
+
+* Agent-based installer
+* Installer-provisioned infrastructure installation
+* User-provisioned infrastructure installation
+
+For more information about installing {product-title} on nodes with dual-port NICs, see xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.html#nw-sriov-dual-nic-con_preparing-to-install-on-bare-metal[NIC partitioning for SR-IOV devices].
```

Comment:
```suggestion
For more information about installing {product-title} on nodes with dual-port NICs, see xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#nw-sriov-dual-nic-con_preparing-to-install-on-bare-metal[NIC partitioning for SR-IOV devices].
```
@kquinn1204 Sorry man. This one got by me the last time.

Reply:
Sorry my bad I should have noticed that @mjpytlak hopefully ok now